### PR TITLE
Fix quoting of values in POST_RECOVERY_SCRIPT and BACKUP_PROG_INCLUDE (Merge SLE-15-SP2 into master)

### PR DESCRIPTION
**Pull request by petrpavlu**
This pull request contains the following:
* Merge branch SLE-15-SP2 into master. This pulls the fix from #29 and commit 76e1e3ae786c8b51631005222b1abfbddf936601.
* Revert 76e1e3ae786c8b51631005222b1abfbddf936601 as it applies only to the SLE-15-SP2 branch (done per https://yastgithubio.readthedocs.io/en/latest/maintenance-branches/#branch-specific-commits).
* Enable unit tests in the CI workflow.

### CODE REVIEW

**Comment by petrpavlu** [comment]
@jreidinger FYI

File: package/yast2-rear.changes [hunk 1]
```diff
@@ -1,8 +1,16 @@
+-------------------------------------------------------------------
+Sat Jan 23 20:14:50 UTC 2021 - Petr Pavlu <petr.pavlu@suse.com>
+
+- Fix quoting of values in POST_RECOVERY_SCRIPT and
+  BACKUP_PROG_INCLUDE (bsc#1180983).
+- 4.2.3
```
Comment (dgdavid) [inline package/yast2-rear.changes:6]: @petrpavlu, @jreidinger 

If I'm not wrong, it should be 4.3.0

**Comment by jreidinger** [comment]
please include also fix from https://github.com/yast/yast-rear/pull/30

File: package/yast2-rear.changes (hunk 1 above)
Comment (jreidinger) [reply package/yast2-rear.changes:6]: yes, as it differs now, it should be 4.3.0

Comment (petrpavlu) [reply package/yast2-rear.changes]: Updated to 4.3.0.

**Comment by petrpavlu** [comment]
Updated pull request fixes the version to 4.3.0 and re-merges SLE-15-SP2 so changes from #30 are included too.

**Review by jreidinger — APPROVED** [review]

**Comment by jreidinger** [comment]
thanks for patch

**Pull request merged**